src/ILCompiler.ReadyToRun/src/Compiler/ProfileData.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
```

Comment:
I think there are unused usings. Would be nice to also group the System.Text with the rest of System.

Reply:
Fixed